### ISSUE TITLE
What is this repos status?

### ISSUE THREAD

**Issue by ff6347** [technologiestiftung/grundsicherung#6]
Since we have grundsicherung_v2 can this be archived?

**Comment by tori-d** [comment]
From my perspective, yes. @julizet can you confirm that this repo is no longer needed since we have grundsicherung_v2?

**Comment by julizet** [comment]
I agree to Tori. grundsicherng_v2 ist a complete copy (+ polyfill) of this repo so "grundsicherung" can be archived
